### PR TITLE
Backport 10009 to 1.5.x

### PR DESCRIPTION
**Pull request by Valarissa**
Backporting an update to the default STS headers.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -78,7 +78,7 @@ require (
 	github.com/hashicorp/vault-plugin-secrets-ad v0.6.6
 	github.com/hashicorp/vault-plugin-secrets-alicloud v0.5.5
 	github.com/hashicorp/vault-plugin-secrets-azure v0.6.2-0.20200818154643-63f769e87b12
-	github.com/hashicorp/vault-plugin-secrets-gcp v0.6.4-0.20200727183837-4005bee700bf
```
Comment (kalafut) [inline go.mod:81]: Why is the GCP dep changing?

Comment (Valarissa) [reply go.mod:81]: I have no idea!

Comment (Valarissa) [reply go.mod]: Ah, I had something committed on the release branch without realizing it. Should be good to go now.

**Review by kalafut — APPROVED** [review]

**Comment by stephane439** [comment]
I'm happy with my own microsolf programm and wondows it my reliability the problem the processor go too fast I find the solution

Téléchargez Outlook pour iOS<https://aka.ms/o0ukef>
________________________________
De : Lauren Voswinkel <notifications@github.com>
Envoyé : Tuesday, September 22, 2020 4:41:19 PM
À : hashicorp/vault <vault@noreply.github.com>
Cc : Subscribed <subscribed@noreply.github.com>
Objet : Re: [hashicorp/vault] Backport 10009 to 1.5.x (#10016)


Merged #10016<https://github.com/hashicorp/vault/pull/10016> into release/1.5.x.

**Pull request merged**